enterprise/frontend/src/embedding-sdk/components/public/InteractiveQuestion/InteractiveQuestion.tsx

```diff
@@ -30,18 +30,23 @@ import {
   type InteractiveQuestionResultProps,
 } from "embedding-sdk/components/private/InteractiveQuestionResult";
 import { withPublicComponentWrapper } from "embedding-sdk/components/private/PublicComponentWrapper";
+import type { SDKCollectionReference } from "embedding-sdk/store/collections";
 
 export type InteractiveQuestionProps = PropsWithChildren<{
   questionId: InteractiveQuestionProviderProps["cardId"];
   plugins?: InteractiveQuestionProviderProps["componentPlugins"];
+  /**
```

Comment:
I manually placed the correct type here because I didn't want to fix the type of the underlying component.
I'm not sure that everywhere the provider is used we support those things and that `InteractiveQuestionProvider` is define in the oss folder which would have required us to move the type.